cmd/main.go

```diff
@@ -294,11 +294,6 @@ func installAutoCompletion() {
 		shellName = filepath.Base(shellName)
 	}
 
-	if completeinstall.IsInstalled(filepath.Base(os.Args[0])) || completeinstall.IsInstalled("mc") {
```

Comment:
> @ebozduman I'm not familiar with Go programming but I'm under the impression that posener/complete v1 doesn't have an API to list shells with completions installed?

@Frederick888,

OK. Then I suggest the change to be as follows.
This way, it addresses both concerns, yours and mine :

```
:
:
	shellName := os.Getenv("SHELL")
	if shellName == "" {
		shellName = "$SHELL"
	} else {
		shellName = filepath.Base(shellName)
	}

	err := completeinstall.Install(filepath.Base(os.Args[0]))
	if err != nil {
		if completeinstall.IsInstalled(filepath.Base(os.Args[0])) || completeinstall.IsInstalled("mc") {
			errStr := err.Error()
			console.Infoln("autocompletion is already enabled in your '"+shellName+"' shell.\n", errStr[strings.Index(errStr, "\n")+1:])
			return
		}
		fatalIf(probe.NewError(err), "Unable to install auto-completion.")
	} else {
		console.Infoln("enabled autocompletion in your '" + shellName + "' rc file. Please restart your shell.")
	}
:
:
```

Reply:
Just pushed 2 commits. First one applied your suggestion. Second one was some extra precautions so that it doesn't confuse users if current shell is not supported. I admit it's ugly though... Feel free to do any modifications or let me know your suggestions.